src/utils/fromPairs.js

```diff
@@ -0,0 +1,22 @@
+/**
```

Comment:
I'd like to avoid this kind of utility here. We can publish it as an node module or use an existing one.